src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/Field.jsx

```diff
@@ -163,11 +164,11 @@ export const FieldCmp = ({classes, inputContext, idInput, selectorType, field, s
                             )}
                         </Grid>
                     </Grid>
-                    <Typography className={classes.errorMessage}>
+                    <Typography className={classes.errorMessage} data-sel-text="fieldErrorMesssage">
```

Comment:
data-sel-text="fieldErrorMesssage" will always be present in the dom even if there is no error.